proto/asgt/type/model.proto

```diff
@@ -6,10 +6,13 @@ option go_package = "type";
 
 import "gen_bq_schema/bq_field.proto";
 import "google/protobuf/timestamp.proto";
+import "asgt/type/target_metrics.proto";
 
 message Model {
   reserved 1 to 2;
   int64 version = 3 [(gen_bq_schema.bigquery).require = true];
-
   google.protobuf.Timestamp created_at = 4;
+  int32 dataset_size = 5;
```

Comment:
Ignore the new metadata field from BQ schema. 

see example: https://github.com/e-conomic/vmlapis/blob/190c9cf9e18aeb1ede5f5af306df7c0bbc055216/proto/ssn/dataservice/v1alpha1/dataservice.proto#L38